components/cdc/src/channel.rs

```diff
@@ -294,15 +297,16 @@ impl Sink {
 }
 
 pub struct Drain {
-    unbounded_receiver: UnboundedReceiver<(CdcEvent, usize)>,
-    bounded_receiver: Receiver<(CdcEvent, usize)>,
+    unbounded_receiver: UnboundedReceiver<(Instant, CdcEvent, usize)>,
+    bounded_receiver: Receiver<(Instant, CdcEvent, usize)>,
```

Comment:
Whether the length of receiver should alse be recorded?

Reply:
Not necessary. I add another metric about sink time in one incremental scan. PTAL.